guide/popular-topics/embeds.md

```diff
@@ -222,15 +218,11 @@ If the images doesn't display inside the embed but outside of it, double check y
 
 At some point in your bot development, you might want to get an embed from a message that you're receiving, edit it, and send it again.
 
-To achieve this, you retrieve the embed from the messages embed array (`message.embeds`) and pass it to the RichEmbed constructor. The constructed RichEmbed can then be edited before sending it again.
-
-::: warning
-You cannot just resend the received embed structure! It's a [MessageEmbed](https://discord.js.org/#/docs/main/stable/class/MessageEmbed) rather than a RichEmbed and contains circular references that prevent sending.
-:::
+To achieve this, you retrieve the embed from the messages embed array (`message.embeds`) and pass it to the MessageEmbed constructor. The constructed MessageEmbed can then be edited before sending it again.
 
 ```js
 const receivedEmbed = message.embeds[0];
-const exampleEmbed = new Discord.RichEmbed(receivedEmbed).setTitle('New title');
+const exampleEmbed = new Discord.MessageEmbed(receivedEmbed).setTitle('New title');
```

Comment:
`receivedEmbed` is already an instance of a MessageEmbed
```suggestion
const exampleEmbed = receivedEmbed.setTitle('New title');
```

Reply:
Per the discussion in discord, going to keep this as-is for consistency and to avoid edge cases.